crow/resources.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: page
+permalink: /crow/resources/
+---
+
+# crow resources
+
+This page will collect links to information and educational resources for crow.
+
+Please feel free to add any resources you would like to see on this page by making a pull requests in the [monome docs repository](https://github.com/monome/docs).
```

Comment:
a pull requests =>

a pull request

Reply:
also, probably best to introduce the abbreviation here; that is:

a pull request (PR)